### DISCUSSION TITLE
How to make menu hidden to some users?

### DISCUSSION THREAD

**Discussion by fromhans** [otto-torino/django-baton#196] [Q&A] [answered]
I wanna make menu hidden to some users.

In Readme.md, it is possible by defining some perms (OR condition) like below.

{ 'type': 'title', 'label': 'main', 'perms': ('auth.add_user', ) },

but i can't understand how to do it with this example.
Can anyone tell me more about this? or any example..

Thank you.

**Comment by abidibo** [comment, marked as answer]
Hi @fromhans the `perms` key in the menu item configuration lets you define a list of permissions. In order to view the menu item the logged in use should have at least one of those permissions. Nothing more than that.
